src/components/Stories.js

```diff
@@ -12,13 +12,13 @@ import userStory from '../services/user_story'
 import ProductList from './ProductList'
 
 const Stories = ({ authorId, followerId }) => {
-  const [currentStateSelected, selectState] = useState('Under consideration')
+  const [currentStateSelected, selectState] = useState('All')
```

Comment:
can we make enum for this instead of string?

Reply:
All the status are already declared in a separate [List](https://github.com/EOS-uiux-Solutions/user-story/blob/main/src/utils/Lists.js#L11) inside the `utils` sub-directory. It behaves like a constant there. Perhaps, instead of the string, I can use `useState(List[0].status)` to use the first element of the list of status i.e `All`.
Should I implement this approach?